ci/da-ghc-lib/compile.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   variables:
     ghc-lib-sha: '77fcc42eef2256ba9bb61a7e442a65bedb19a35d'
     base-sha: '9c787d4d24f2b515934c8503ee2bbd7cfac4da20'
-    patches: 'f844cb345412f6a9fd5708d8b697fd0800dd0909 833ca63be2ab14871874ccb6974921e8952802e9'
+    patches: '80f2ae67ec69d79c116b1e239228635dc460e783 833ca63be2ab14871874ccb6974921e8952802e9'
```

Comment:
I’ll bump this again once the PR to our GHC fork is merged but I wanted to see if CI passes.